vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -621,7 +622,8 @@ base_learner* ccb_explore_adf_setup(options_i& options, vw& all)
                .necessary()
                .help(
                    "EXPERIMENTAL: Do Conditional Contextual Bandit learning with multiline action dependent features."))
-      .add(make_option("all_slots_loss", all_slots_loss_report).help("Report average loss from all slots"));
+      .add(make_option("all_slots_loss", all_slots_loss_report).help("Report average loss from all slots"))
+      .add(make_option("ccb_model_without_has_seen_flag", data->input_model_without_has_seen_multi_slot_flag).help("Legacy ccb model format without HasSeenMultipleSlots flag"));
```

Comment:
Is this options meant to be stored in the model? Should it be 'keep'?

Reply:
I do not think so. Option is used only in scenario: --ccb_explore_adf -i cb_model.bin
Since we are doing --ccb_explore_adf output model is ccb model, and this flag is not applicable there.

Reply:
sorry.
other scenario is backward compatibility one - when we are running new vw in ccb mode using old ccb model.
I still feel that it is better that model should be produced in new format (since it is new vw), but it seems a bit open question actually.